packages/app/src/systems/Account/machines/accountsMachine.tsx

```diff
@@ -103,6 +109,13 @@ export const accountsMachine = createMachine(
           UPDATE_ACCOUNT: {
             target: 'updateAccount',
           },
+          EDIT_ACCOUNT: {
+            actions: ['assignAddress'],
```

Comment:
When doing the action to assign the address, it would be good for the machine to transit to a state fetching account using the address informed, after that the account for edit will be on the machine context. If it is not found, a tooltip will show automatically as it uses the fetchMachine.

This will remove the logic inside the edit page, where you do a find on the accounts after the page is open.